modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -820,7 +820,7 @@
     </xsd:complexType>
 
 
-    <xsd:element name="watermark" type="watermarkType" substitutionGroup="abstract-poll-override">
+    <xsd:element name="watermark" type="watermarkType" substitutionGroup="abstract-scheduler-override">
```

Comment:
Why is this still in schema if watermark was removed?  Do we still need scheduler overrides even now?